docs/_user_guide/best_practices_python_env.md

```diff
@@ -1,153 +1,171 @@
 
-```bash
-$ cd <path/to/project>
+```title="Terminal Command"
+cd <path/to/project>
 ```
 where `<path/to/project>` is the shared project location on ZFS.
 
-Create a new virtual environment:
+Create a new virtual environment named `my_env` using the system Python path. This will create a directory called `my_env` inside the project directory:
 
-```bash
-$ /usr/bin/python3  -m venv .venv
+```title="Terminal Command"
+/usr/bin/python3  -m venv my_env
 ```
-where we make a hidden directory `.venv` inside the project directory. 
 
-## Activating a New Virtual Environment 
+## Activating a New Virtual Environment
 
 Next, we activate the virtual environment:
-```bash
-$ source .venv/bin/activate
+
+```title="Terminal Command"
+source my_env/bin/activate
 ```
 
-You should see `(.venv):` prepended to the prompt: 
-```bash
-(.venv): 
+You should see `(my_env):` prepended to the prompt:
+
+```{ .yaml .no-copy title="Terminal Output" }
+(my_env):
 ```
 
 Check Python version:
 
+```title="Terminal Command"
+python --version
 ```
-$ python --version
-Python 3.10.12
+```{ .yaml .no-copy title="Terminal Output" }
+python 3.10.12
 ```
 
 ## Installing Python Packages into the New Virtual Environment
-Install any python package with `pip`:
-
-```bash
-$ pip install <package>
-```
 
-where `<package>` is a Python package (or list) to install, such as `numpy`, `pandas`, etc.
+Install any Python package with `pip`:
 
-## Making the Virtual Environment into a JupyterHub Kernel 
-Install `ipykernel` package before installing the new environment as a kernel on JupyterHub:
-
-```bash
-$ pip install ipykernel
+```title="Terminal Command"
+pip install <package>
 ```
 
-To add the **active** virtual environment as a kernel, run:
-```bash
-$ python -m ipykernel install --user --name=<kernel-name>
-```
-where `<kernel-name>` is the name of the kernel on JupyterHub.
+where `<package>` is a Python package (or list) to install, such as `numpy` or `pandas`.
 
 ## Running Python Scripts Using Virtual Environment
-Using your environment is very simple - as long as your environment is activated, you can run python normally:
 
-```
-(.venv) USER@yenX:$ python <my_script.py> 
+Using your environment is straightforward—as long as your environment is activated, you can run Python as usual:
+
+```title="Terminal Command"
+python <my_script.py>
 ```
 where `<my_script.py>` is your Python script.
 
-The Python command will be specific to your environment. You can troubleshoot this with the `which` command:
+The Python executable will be specific to your environment. You can troubleshoot this with the `which` command:
 
+```title="Terminal Command"
+which python
 ```
-(.venv) USER@yenX:$ which python
+```{ .yaml .no-copy title="Terminal Output" }
 /path/to/env/bin/python
 ```
-where `/path/to/env/bin/python` is the path to the Python in your environment.
+where `/path/to/env/bin/python` is the path to the system Python in your environment.
```

Comment:
I would delete "system" since it is not the system python anymore but a python binary inside your env